src/components/TrashNote/TrashNote.jsx

```diff
@@ -2,75 +2,100 @@ import React, { useState } from "react";
 import { Link } from "react-router-dom";
 import { useNote } from "../../contexts/Note/NoteContext";
 import styles from "./TrashNote.module.css";
-
+import DOMPurify from "dompurify";
 export default function TrashNote({ note }) {
-  const { title, notes } = note;
+  const { title, notes, timestamp, color, label } = note;
   const [options, setOptions] = useState(false);
-  const { noteState, setNote, noteDispatch, archiveNote, RestoreArchivedNote } =
-    useNote();
-
-  const { color, label, day, timestamp } = note;
+  const {
+    noteState,
+    noteDispatch,
+    deleteNote,
+    restoreArchivedNote,
+    deleteArchivedNote,
+  } = useNote();
 
-  const deleteFromTrash = (note) => {
-    noteDispatch({ type: "DELETE_FROM_TRASH", payload: note });
-  };
   const optionsHandler = () => {
     setOptions((prev) => !prev);
   };
 
+  const deleteHandler = (note) => {
+    deleteNote(note, noteDispatch);
+    noteState.archivedNotes.filter((item) => item._id !== note._id);
+  };
+
   return (
     <div
-      className={`${styles.note} border-radius box-shadow position-relative`}
+      className={`${styles.note} ${
+        options ? styles.enlargeNote : styles.defaultNote
+      } border-radius box-shadow position-relative`}
       style={{ backgroundColor: `${color}` }}
     >
-      <div onClick={optionsHandler}>
-        <span className={`position-absolute  ${styles.options} material-icons`}>
-          more_horiz
-        </span>
-      </div>
       <div className={`${styles.note_content} flex-column-wrap flex-top-left`}>
-        <h3 className={styles.note_title}>{title}</h3>
+        <h3 vs className={styles.note_title}>
+          {title}
+        </h3>
         <p
           className={styles.note_body}
           dangerouslySetInnerHTML={{
-            __html: notes,
+            __html: DOMPurify.sanitize(notes),
```

Comment:
This looks new. Will explore more on it!!

Reply:
Yes, it plays an important role, feel free to ping me if you need more clarity on this.